artiq/firmware/runtime/rtio_mgt.rs

```diff
@@ -212,11 +212,11 @@ pub mod drtio {
                                 destination_set_up(routing_table, up_destinations, destination, false),
                             Ok(drtioaux::Packet::DestinationOkReply) => (),
                             Ok(drtioaux::Packet::DestinationSequenceErrorReply { channel }) =>
-                                error!("[DEST#{}] RTIO sequence error involving channel 0x{:04x}", destination, channel),
+                                unsafe{SEEN_ASYNC_ERRORS |= 4},
```

Comment:
Those numbers are also used here:
https://github.com/m-labs/artiq/blob/9b0bdd1ce32666ff928b2a52953298409118b613/artiq/firmware/runtime/rtio_mgt.rs#L160-L168
Define some ``const``

Reply:
And we still want the ``error!`` messages, the "seen errors" at the end of kernel execution is just an additional (and imprecise) indicator as discussed in https://github.com/m-labs/artiq/pull/1889.